Source/WebCore/svg/SVGLengthContext.cpp

```diff
@@ -272,7 +272,23 @@ float SVGLengthContext::computeNonCalcLength(float inputValue, CSS::LengthUnit u
         auto conversionData = cssConversionData();
         if (!conversionData)
             return 0.0f;
-        return clampTo<float>(Style::computeNonCalcLengthDouble(inputValue, unit, *conversionData));
+
+        auto resolvedValue = clampTo<float>(Style::computeNonCalcLengthDouble(inputValue, unit, *conversionData));
+
+        if (unit == CSS::LengthUnit::Em || unit == CSS::LengthUnit::Rem) {
+            auto* rootStyle = conversionData->rootStyle();
+            auto& fontCascade = (unit == CSS::LengthUnit::Rem && rootStyle)
+                ? rootStyle->fontCascade()
+                : conversionData->fontCascadeForFontUnits();
+            auto& fontDescription = fontCascade.fontDescription();
+
+            float computedSize = fontDescription.computedSize();
```

Comment:
Is this guaranteed to be unclamped? I vaguely recall some code to clamp the font size above a minimum value.

Alternatively, you could directly access the effective zoom, no?

Reply:
> Is this guaranteed to be unclamped? I vaguely recall some code to clamp the font size above a minimum value.

```
fontDescription.setComputedSize(Style::computedFontSizeFromSpecifiedSize(size, fontDescription.isAbsoluteSize(), useSVGZoomRules(), &style(), document()));
```

are you referring to `computedFontSizeFromSpecifiedSize`? at style-resolution time